kobo/apps/superuser_stats/views.py

```diff
@@ -45,10 +45,10 @@ def country_report(request):
 
     # Get the date filters from the query and set defaults
     if not (start_date := request.GET.get('start_date')):
-        start_date = f'{today}'
+        start_date = str(first_of_month)
 
     if not (end_date := request.GET.get('end_date')):
-        end_date = f'{today}'
+        end_date = str(today)
```

Comment:
ok, looks like `__date__range` is inclusive, so this should work even `today == first_of_month` :+1: